scala-protoc-plugins/scala-akka/AkkaGrpcServicePrinter.scala

```diff
@@ -71,8 +71,7 @@ final class AkkaGrpcServicePrinter(
       .add(s"protected val killSwitch = akka.stream.KillSwitches.shared($killSwitchName)")
       .add("protected val closed = new java.util.concurrent.atomic.AtomicBoolean(false)")
       .add(
-        // TODO error code: Use self service error code
-        "protected def closingError = new io.grpc.StatusRuntimeException(io.grpc.Status.UNAVAILABLE.withDescription(\"Server is shutting down\")) with scala.util.control.NoStackTrace"
+        "protected def closingError = com.daml.grpc.adapter.server.akka.ServerAdapter.closingError()"
```

Comment:
Delegating to `erverAdapter.closingError` which in turn is creating an error codes' based exception